README.md

```diff
@@ -0,0 +1,241 @@
+# Application Folder
+
+## Contents
+
+- General
+- Directory Location
+- Built-In Applications
+- NuttShell (NSH) Built-In Commands
+- Synchronous Built-In Commands
+- Application Configuration File
+- Example Built-In Application
+- Building NuttX with Board-Specific Pieces Outside the Source Tree
+
+## General
+
+This folder provides various applications found in sub-directories. These
+applications are not inherently a part of NuttX but are provided to help
+you develop your own applications. The `apps/` directory is a _break away_
+part of the configuration that you may choose to use or not.
+
+## Directory Location
+
+The default application directory used by the NuttX build should be named
+`apps/` (or `apps-x.y.z/` where `x.y.z` is the NuttX version number). This `apps/`
+directory should appear in the directory tree at the same level as the
+NuttX directory. Like:
+
+```
+ .
+ |- nuttx
+ |
+ `- apps
```

Comment:
It is easier and simpler to just prepend these kind of verbatim blocks with four spaces. The backticks are not required (unless you need to escape something I think)

Reply:
The indented form doesn't allow selecting language for syntax highlighting. We could use indented for non-highlighted fragments and backticks for highlighted fragments. But I think it is better to stick to one.

Also, as far as I remember I always see only the backtick form being used. Example: https://raw.githubusercontent.com/ARMmbed/littlefs/master/README.md

Just for consistency I would stick to triple backtick everywhere.